tests/attestation_plugin_cert/tls.edl

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/logging.edl" import *;
+    from "openenclave/edl/fcntl.edl" import *;
+#ifdef OE_SGX
+    from "openenclave/edl/sgx/platform.edl" import *;
+#else
+    from "openenclave/edl/optee/platform.edl" import *;
+#endif
+
+    trusted {
+        public oe_result_t get_tls_cert_signed_with_ec_key([out] unsigned char** data, [out] size_t* data_size);
+        public oe_result_t get_tls_cert_signed_with_rsa_key([out] unsigned char** data, [out] size_t* data_size);
```

Comment:
```suggestion
        public oe_result_t get_tls_certificate_signed_with_ec_key([out] unsigned char** data, [out] size_t* data_size);
        public oe_result_t get_tls_certificate_signed_with_rsa_key([out] unsigned char** data, [out] size_t* data_size);
```

Reply:
Will keep to be compatible with existing code base.